### PR TITLE
Style theme settings editor

### PR DESCRIPTION
**Pull request by glebovsky**
image

Gradient picker:
image

### CODE REVIEW

File: src/common/components/atoms/button.tsx [hunk 1]
```diff
@@ -11,13 +11,14 @@ const buttonVariants = cva(
...
           "bg-primary text-primary-foreground shadow hover:bg-primary/90",
+        primary: "rounded-xl text-white bg-blue-500 hover:bg-blue-600",
         destructive:
           "bg-destructive text-destructive-foreground shadow-sm hover:bg-destructive/90",
         outline:
           "border border-input bg-transparent shadow-sm hover:bg-accent hover:text-accent-foreground",
         secondary:
-          "bg-secondary text-secondary-foreground shadow-sm hover:bg-secondary/80",
-        ghost: "hover:bg-accent hover:text-accent-foreground",
+          "bg-secondary text-[#1C64F2] rounded-xl shadow-sm hover:bg-sky-100",
```
Comment (nounspaceryan) [inline src/common/components/atoms/button.tsx:20]: Can we keep text-secondary-foreground and change its tailwind value instead? Same for hover:bg

File: src/common/components/atoms/button.tsx [hunk 2]
```diff
@@ -11,13 +11,14 @@ const buttonVariants = cva(
       variant: {
         default:
           "bg-primary text-primary-foreground shadow hover:bg-primary/90",
+        primary: "rounded-xl text-white bg-blue-500 hover:bg-blue-600",
```
Comment (nounspaceryan) [inline src/common/components/atoms/button.tsx:14]: Thinking we should move "rounded-xl" to the size.default object. same for secondary variant

File: src/common/components/atoms/button.tsx (hunk 1 above)
Comment (glebovsky) [reply src/common/components/atoms/button.tsx:20]: Sure! will do!

File: src/common/components/atoms/button.tsx (hunk 2 above)
Comment (glebovsky) [reply src/common/components/atoms/button.tsx:14]: Gocha! will do!

**Review by willyogo — APPROVED** [review]

**Pull request merged**